specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/SessionPools.json

```diff
@@ -404,7 +404,7 @@
       "description": "Session pool private registry credentials.",
       "type": "object",
       "properties": {
-        "registryServer": {
+        "server": {
```

Comment:
I think you need to update 2024-10-02-preview.

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/30771